jobs/fatal_od.py

```diff
@@ -40,8 +40,22 @@ def combine_date_and_time(self, in_data):
         )
         return
 
+    @post_load
```

Comment:
Well marshmallow is a [serialization library](http://marshmallow.readthedocs.org/en/latest/quickstart.html#serializing-objects-dumping) as well, so we should be able to take advantage of this.